protocol-units/da/m1/setup/src/local.rs

```diff
@@ -1,16 +1,15 @@
+use crate::M1DaLightNodeSetupOperations;
+use celestia_types::nmt::Namespace;
 use commander::run_command;
 use dot_movement::DotMovement;
+use rand::Rng;
```

Comment:
Nitpick: I like to group use statements by dependencies in the same namespace, external crates, and `std`/`core`.

Reply:
I'm used to remove all blank line in use statement because rustfmt sort them automatically if there are all in the same block. Sorted use is very convenient when you search for one. But I can change and keep your separate block.

Reply:
Yes, sorting within groups is fine, but I like to keep the imports grouped by provenance as well.